server/schemas/types/expense.type.js

```diff
@@ -0,0 +1,39 @@
+import {
+  GraphQLString,
+  GraphQLFloat,
+  GraphQLObjectType,
+  GraphQLID,
+  GraphQLNonNull
+} from 'graphql';
+import GraphQLDate from 'graphql-date';
+import CategoryType from './category.type';
+
+export default new GraphQLObjectType({
+  name: 'ExpenseType',
+  fields: {
+    id: {
+      type: GraphQLID,
+      description: 'ID of expense',
+    },
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Name of expense',
+    },
+    category: {
+      type: new GraphQLNonNull(CategoryType),
+      description: 'Category of expense',
+    },
+    amount: {
+      type: new GraphQLNonNull(GraphQLFloat),
+      description: 'Amount of expense',
+    },
+    createdAt: {
```

Comment:
Is it possible that these have default values? `createdAt` and `updatedAt`I mean, so that it removes the overhead of having to manually input values there? Something like `Date.now()` blabla.

Reply:
This is the GraphQL schema for the query. Mongoose already inputs these values for us in the expense model @oreoluwade

Reply:
Awesome!